.github/generators/docs_generator.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+from pathlib import Path
+from jinja2 import Template
+import os, argparse, json
+
+severities = {'High': 'HIGH', 'Medium': 'MEDIUM', 'Low': 'LOW', 'Info': 'INFO'}
+colors = {'High': '#C00', 'Medium': '#C60', 'Low': '#CC0', 'Info': '#00C'}
+template_dict = {}
+platforms = []
+
+arg_parser = argparse.ArgumentParser(description='Create query docs')
+arg_parser.add_argument('-p', nargs=1, type=Path, dest='intput_path', help='Path to read metadata.json')
+arg_parser.add_argument('-o', nargs=1, type=Path, dest='output_path', help='Path to output documentation files')
+arg_parser.add_argument('-t', nargs=1, type=Path, dest='templates_path', help='Path to template files')
+arg_parser.add_argument('-f', nargs='+', type=str, dest='formats', help='Formats to be created documentation, must exists on template folder')
```

Comment:
`Documentation formats to be created, the extension must exists in the template folder`